plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -492,7 +492,132 @@ def download_path(self):
 
 
 class RpmBase(NonMetadataPackage):
-    # TODO add docstring to this class
+    """
+    This class is designed to be sub-classed by both RPM and SRPM, as these two
+    package types are similar. Most fields map to metadata fields in the RPM package
+    format.
+
+    :ivar arch: The target architecture for a package. For example, 'x86_64', 'i686',
+                or 'noarch'.
+    :type arch: mongoengine.StringField
+
+    :ivar buildhost: Hostname of the system that built the package.
+    :type buildhost: mongoengine.StringField
+
+    :ivar changelog: A list of changelog entries for the package. What these changelogs
+                     are used for depend on the packager, but Fedora uses to changelog
```

Comment:
The second half needs to be a complete sentence or the comma needs to be removed from before the `but` since it's not being used as a conjunction. I recommend changing s/uses to changelog/uses this field for changelogs/